lib/server/readMetadata.js

```diff
@@ -81,7 +81,8 @@ function readSidebar() {
 
 // split markdown header
 function splitHeader(content) {
-  const lines = content.split('\n');
+  // New line characters need to handle all operating systems.
+  const lines = content.split(/\r?\n/);
```

Comment:
👍 

Nice fix.

Reply:
So this will catch both `\r\n` and `\n`. I assume we don't have to worry about just `/r` anywhere?

Reply:
I tested it with `/r`,`/n` as basic text and it just matches the carriage return characters.

/r/n is for Windows
/n is for UNIX systems

According to https://en.wikipedia.org/wiki/Newline , if we have any pre OSX users, they would have the /r carriage return.